frontend/src/views/main/project/TestSuites.vue

```diff
@@ -177,14 +181,19 @@ onActivated(async () => {
     }
 })
 
-function renameSuite(suiteId: number, name: string) {
+async function renameSuite(suiteId: number, name: string) {
     const currentSuite = testSuitesStore.testSuites.find(s => s.id === suiteId);
 
     if (currentSuite) {
         currentSuite.name = name;
-        testSuitesStore.updateSuiteName(currentSuite);
-
-        mixpanel.track('Renamed test suite', {suiteId});
+        await testSuitesStore.updateSuiteName({
```

Comment:
why do you have to create a new object instead of sending a `currentSuite` with updated name?

Reply:
For some reason, it doesn't update the name on the list after sending currentSuite. This was the alternative I found to make automatically update.

Reply:
I'm gonna test if this behaviour happens again if I use `currentSuite`

Reply:
Now it's working with `currentSuite`. Just pushed it.